gravitee-apim-console-webui/src/components/gio-side-nav/gio-side-nav.component.ts

```diff
@@ -53,25 +51,25 @@ export class GioSideNavComponent implements OnInit {
 
   constructor(
     @Inject(UIRouterState) private readonly ajsState: StateService,
-    @Inject(AjsRootScope) private readonly ajsRootScope: IRootScopeService,
     private readonly permissionService: GioPermissionService,
-    private readonly environmentService: EnvironmentService,
     @Inject(PortalSettingsService) private readonly portalConfigService: PortalConfigService,
     @Inject(CurrentUserService) private readonly currentUserService: UserService,
     @Inject('Constants') private readonly constants: Constants,
     private readonly gioLicenseService: GioLicenseService,
   ) {}
 
   ngOnInit(): void {
-    this.mainMenuItems = this.buildMainMenuItems();
-    this.footerMenuItems = this.buildFooterMenuItems();
-    this.environments = this.constants.org.environments.map((env) => ({ value: env.id, displayValue: env.name }));
+    this.gioLicenseService.isOEM$().subscribe((isOEM) => {
```

Comment:
Does `.isOEM$` call the back every time or it just uses the License stored in the Observable in Particles?

Reply:
No it's a call to the backend. License can be hot reloaded so I guess we need to keep this behavior (even if it doesn't change so often)